<ui version="4.0" >
 <author>fullmetalcoder</author>
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>588</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Configuration</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabOptions" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>658</width>
        <height>504</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Global Preferences</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Language</string>
            </property>
            <property name="buddy" >
             <cstring>cboLanguage</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Perspective</string>
            </property>
            <property name="buddy" >
             <cstring>cboPerspective</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="cboLanguage" >
            <property name="currentIndex" >
             <number>1</number>
            </property>
            <item>
             <property name="text" >
              <string>Untranslated</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>System locale</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Last Used</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Instances</string>
            </property>
            <property name="buddy" >
             <cstring>cboInstances</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="cboPerspective" >
            <property name="currentIndex" >
             <number>1</number>
            </property>
            <property name="sizeAdjustPolicy" >
             <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
            </property>
            <item>
             <property name="text" >
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Last Used</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QComboBox" name="cboInstances" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <property name="sizeAdjustPolicy" >
             <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
            </property>
            <item>
             <property name="text" >
              <string>Single</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Multiple</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QCheckBox" name="cbWarnUponMinimize" >
            <property name="text" >
             <string>Warn upon minimization</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QCheckBox" name="cbMinimizeToTray" >
            <property name="text" >
             <string>Minimize to system tray</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Manager dock display mode</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QComboBox" name="cboManagerDock" >
            <item>
             <property name="text" >
              <string>Tabs</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Toolbox</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Combobox</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>On Edyuk Start</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <widget class="QCheckBox" name="chkReopenFiles" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Reopen files?</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="cboReopenProjects" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <item>
             <property name="text" >
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Current</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="cboReopenFiles" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <item>
             <property name="text" >
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Current</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QCheckBox" name="chkReopenProjects" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Reopen projects?</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Most Recently Used</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="2" >
           <widget class="QSpinBox" name="spnDisplayRecentProjects" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>20</number>
            </property>
            <property name="value" >
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QLabel" name="label_11" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>recent files.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLabel" name="label_7" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>recent projects.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QSpinBox" name="spnDisplayRecentFiles" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>30</number>
            </property>
            <property name="value" >
             <number>15</number>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="0" column="4" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>55</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QLabel" name="label_8" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Display</string>
            </property>
            <property name="buddy" >
             <cstring>spnDisplayRecentFiles</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QLabel" name="label_6" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Display</string>
            </property>
            <property name="buddy" >
             <cstring>spnDisplayRecentProjects</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Auto save</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Before build</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="cboAutoSaveOnBuild" >
            <item>
             <property name="text" >
              <string>Always ask</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Save without asking</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Discard changes</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Before exit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="cboAutoSaveOnExit" >
            <item>
             <property name="text" >
              <string>Always ask</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Save without asking</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Discard changes</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <tabstops>
  <tabstop>tabOptions</tabstop>
 </tabstops>
 <resources>
  <include location="../images/Edyuk.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cbMinimizeToTray</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbWarnUponMinimize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>103</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>386</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
